build.sbt

```diff
@@ -10,8 +10,12 @@ resolvers += "Akka Snapshot Repository" at "http://repo.akka.io/snapshots/"
       
 scalaVersion := "2.12.2"
 
-libraryDependencies ++= Seq( jdbc , ehcache , ws , specs2 % Test , guice )
+libraryDependencies += guice
+libraryDependencies ++= Seq( ehcache , ws )
```

Comment:
do we need it? this wasn't in gowno repository

Reply:
Right, it's probably a better idea to add dependencies as they become needed. Removed.